lib/csv/parser.rb

```diff
@@ -47,7 +47,7 @@ class UnexpectedError < StandardError
     # Uses StringScanner (the official strscan gem). Strscan provides lexical
     # scanning operations on a String. We inherit its object and take advantage
     # on the methods. For more information, please visit:
-    # https://ruby-doc.org/stdlib-2.6.1/libdoc/strscan/rdoc/StringScanner.html
+    # https://docs.ruby-lang.org/en/3.4/StringScanner.html
```

Comment:
How about using "master" here? We don't want to maintain the latest version...

```suggestion
    # https://docs.ruby-lang.org/en/master/StringScanner.html
```

Reply:
FYI: This is an internal class. (This is not for users.) So we can use "master" here.